de-de/ressourcenpaket-probleme.md

```diff
@@ -0,0 +1,87 @@
+# Probleme mit Ressourcenpaketen
+
+*Fabulously Optimized* unterstützt [einige Funktionen von Optifine](verzichte-auf-optifine.md), darunter auch Extras für Ressourcenpakete. Allerdings kann es hierbei auch Probleme geben. Dieses Tutorial zeigt dir, wie du sie löst!
+
+Bitte [stelle sicher, dass du die neuste Version von *Fabulously Optimized* nutzt](update-anleitung.md), bevor du versuchst, die Probleme zu beheben.
+
+### Falsche Dateipfade
+
+Siehst du diese Fehlermeldung bei einem Ressourcenpaket?
+
+![Contains broken paths](https://i.ibb.co/26cMtqr/Screenshot-20211116-191457.png)
+
+Das heißt, dass der Ersteller des Ressourcenpakets Sonderzeichen in Ordner- oder Dateinamen hat.
+
+Falls dies der Fall ist, wird das Modpack versuchen diese zu überschreiben. Das kann allerdings zu eine menge Fehler führen(Blöcke mit den falschen Farben oder unsichtbare Blöcke, Probleme mit der Beleuchtung, kaputte Texturen...).
+
+Deswegen solltest du den Ersteller des Ressourcenpakets bitten, nur folgende Zeichen in Datei- und Ordnernamen zu verwenden: `a-z`, `0-9` und `._-`.
+
+### Unsichtbare Blöcke
+
+Dieses Problem kann auftreten, wenn das Ressourcenpaket falsche Pfade verwendet und versucht, die Modelle von Blöcken wie z.B. Truhen zu verändern.
+
+**Enhanced Block Entities**
+
+*Enhanced Block Entities* ist eine Mod in *Fabulously Optimized*, welche das Rendering von Böcken optimiert. Allerdings werden manche Optimierungen von deinem Ressourcenpaket nicht unterstützt.
+
+So kannst du das Problem lösen:
+
+1. Öffne im Spielmenü `Mods`
+2. Suche nach "Enhanced Block Entities" und drücke das Einstellungssymbol ![](https://i.ibb.co/j35cBtn/image.png)
+3. Stelle alle Blöcke auf "aus", die Probleme verursachen. Meistens genügt nur `Enhanced Chests`.
+4. Drücke `Done`
```

Comment:
You should check whether Mod Menu and those mods are already translated in German, e.g. I don't think it should say "Done" here

Reply:
As English is the default language, I didn't translate all the buttons in the wiki.¹

Another reason why I did this is because sometimes, the translations for other languages get changed and the text doesn't make sense anymore. On the other hand, some aren't translated at the moment at all...

¹ Also, I'd be quite a lot of effort installing ***all*** launchers mentioned in the wiki and checking through every single menu 😅... (of course, some of the Mods are open source and the translations can be looked up but others aren't...). So, a third reason against translating the menus would be consistency.

Reply:
Fair enough.